pkg/logger/log/accesslog.go

```diff
@@ -43,7 +43,9 @@ var (
 )
 
 const (
-	fullAccessLogFormat = `time=%s log_type=access method=%s path="%s" status=%d duration=%d length=%d source_ip=%s user_agent="%s" referer="%s" trace_id=%s namespace=%s user_id=%s client_id=%s request_content_type="%s" request_body=AB[%s]AB response_content_type="%s" response_body=AB[%s]AB operation="%s" flight_id="%s" game_version="%s" sdk_version="%s" oss_version="%s"`
+	fullAccessLogFormat = `time=%s log_type=access method=%s path="%s" status=%d duration=%d length=%d source_ip=%s user_agent="%s" referer="%s" trace_id=%s namespace=%s user_id=%s client_id=%s request_content_type="%s" request_body=AB[%s]AB request_body_size_in_kb=%.3f response_content_type="%s" response_body=AB[%s]AB response_body_size_in_kb=%.3f operation="%s" flight_id="%s" game_version="%s" sdk_version="%s" oss_version="%s"`
```

Comment:
could we just use `request_body_size` and `response_body_size` to keep it smaller, and save cost as well

Reply:
hi, what about `req_body_size_kb` and `resp_body_size_kb`?  they are still short and still tell the measurement unit

Reply:
yes, both is fine, the goal is to shorten the logs